pom.xml

```diff
@@ -565,7 +565,7 @@
             <id>spring-boot-2.0</id>
             <properties>
                 <spring.version>5.0.4.RELEASE</spring.version>
-                <spring-boot.version>2.0.0.RC2</spring-boot.version>
+                <spring-boot.version>2.0.0.RELEASE</spring-boot.version>
             </properties>
             <repositories>
```

Comment:
Can you also remove the `repositories` section? We shouldn't need it anymore, since it's now in central.